django_app/redbox_app/redbox_core/models.py

```diff
@@ -619,7 +619,7 @@ def get_ordered_by_citation_priority(cls, chat_message_id: uuid.UUID) -> Sequenc
         )
 
     def position_in_queue(self) -> int:
-        if not self.task:
+        if not self.task_id:
```

Comment:
`.task` is the object itself, this will error if it doesnt exist, whereas `.task_id` is just the `id` which can be null